src/tactic/tidy.lean

```diff
@@ -80,7 +80,11 @@ namespace interactive
 open lean.parser interactive
 
 /-- Use a variety of conservative tactics to solve goals.
-`tidy?` reports back the tactic script it found. -/
+`tidy?` reports back the tactic script it found.
+The default list of tactics is stored in `default_tidy_tactics`.
+This list can be overridden using `tidy { tactics := ... }`.
+(The list must be a `list` of `tactic string`, so that `tidy?`
+can report a usable tactic script.) -/
```

Comment:
Should we include a pointer to more documentation? For example it seems helpful to know that one can (locally) add the `tidy` attribute to tactics to add them to the default list. But not sure if that should be in the docstring...

Reply:
Oh, that's a good idea. I think it's OK to put it in the docstring. For instance, the `simp` docstring is quite detailed and I've found it pretty helpful.